dlt/destinations/sql_jobs.py

```diff
@@ -582,8 +579,22 @@ def gen_scd2_sql(
                     staging_table_name = sql_client.make_qualified_table_name(table["name"])
                 sql.append(f"""
```

Comment:
Could this SQL statement be the following, for consistency with the logic for the root table?

```sql
  INSERT INTO {table_name}
  SELECT *
  FROM {staging_table_name}
  WHERE {unique_column} NOT IN (SELECT {unique_column} FROM {table_name});
```

Reply:
works! thanks!